course-project/Backend/src/cmd/rest-api/handlers/list.go

```diff
@@ -0,0 +1,24 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/lunarnuts/go-course/tree/course-project/course-project/Backend/src/cmd/rest-api/lib"
+	"github.com/lunarnuts/go-course/tree/course-project/course-project/Backend/src/db/db"
+	records "github.com/lunarnuts/go-course/tree/course-project/course-project/Backend/src/db/models"
+)
+
+func SelectAll(p *pgxpool.Pool, w http.ResponseWriter, r *http.Request) {
+	conn, err := db.AcquireConn(p)
+	if err != nil {
+		lib.ReturnInternalError(w, err)
+		return
+	}
+	recs, err := records.SelectAll(conn)
```

Comment:
Forgot to mention first time. If the list is empty the endpoint returns just "null", while it is preferable to receive and empty json array, since the client, that is going to consume it, will be expecting that, instead of a rando null string

Reply:
fixed it by initializing recs with make() function